doors_dashboards/components/meteogram.py

```diff
@@ -10,6 +10,8 @@
 from typing import Tuple
 import dash_bootstrap_components as dbc
 
+from doors_dashboards.components.constant import FONT_FAMILY, FONT_COLOR, PLOT_BGCOLOR, \
+    FONT_SIZE, FONT_SIZE_NUMBER
```

Comment:
```suggestion
from doors_dashboards.components.constant import FONT_COLOR
from doors_dashboards.components.constant import FONT_FAMILY
from doors_dashboards.components.constant import FONT_SIZE
from doors_dashboards.components.constant import FONT_SIZE_NUMBER
from doors_dashboards.components.constant import PLOT_BGCOLOR
```
The line was a bit too long. Also, we somewhere had a rule that we want to avoid multiple inputs per line. Finally, I like when they are ordered alphabetically, but that is a matter of taste.